app/views/menus/show.html.haml

```diff
@@ -1,8 +1,13 @@
 %h2 Menu
-
-=@restaurant.name
-=@restaurant.menu.name
-
--@restaurant.menu.dishes.each do |dish|
-  =dish.name
-  =dish.price
+= notice
+= @cart_nr
+dishes in cart
+= @restaurant.name
+= @restaurant.menu.name
+- @restaurant.menu.dishes.each do |dish|
+  %div{class: "dish-#{dish.id}"}
+    = dish.name
+    = dish.price
+    = form_tag(items_path, method: 'post') do
+      = submit_tag('Buy')
+      = hidden_field_tag(:dish_id, dish.id)
```

Comment:
This is ok for now but it is hideously ugly. You may want to use some basic headers or div separators here. You're going to clean things up later but it's good to leave views in a state that is at least understandable.